### ISSUE TITLE
[Bug] Won't show album titles when playing from library on Apple Music?

### ISSUE THREAD

**Issue by HJ2343** [ungive/discord-music-presence#85]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Describe the bug

So, when I play music from my library (Apple Music) not downloaded ones, but actual ones just added from my subscription the album covers won't play.  If I search for them on Apple Music and play them they will show the album covers.

### Reproduction Steps

1. Play Music from saved in library
2. No album cover

### Expected behavior

An album cover shown, not the Apple Music logo

### Device Info

- Discord Music Presence version: [newest
- Operating system: [Mac]
- The media player you used (if applicable): [Apple Music]

**Comment by ungive** [comment]
Can you check if there's a cover in the media control menu of your device: https://imgur.com/a/YDQegwW

And could you also attach your logs? You can find them here: `~/Library/Application Support/Music Presence/presence.log`

**Comment by HJ2343** [comment]
There is a cover in my media control menu.  It shows not only what I'm playing in Apple Music, but also my Podcast App.

When I goto my Library/Application Support Music Presence isn't an available folder?

**Comment by ungive** [comment]
> There is a cover in my media control menu. It shows not only what I'm playing in Apple Music, but also my Podcast App.

Thanks for the info.

> When I goto my Library/Application Support Music Presence isn't an available folder?

It's located in the Library folder that is located in your Home folder, it's a bit hidden: https://www.autodesk.com/support/technical/article/caas/sfdcarticles/sfdcarticles/How-to-Access-Hidden-User-Library-folder-on-Mac-OS.html

This issue might be related: #77.

If you see this in your logs, it's most likely the bug mentioned in that issue:

```
ERROR Using cover image from player failed: Failed to convert image for proxy: Conversion result is empty
```

There's a build on our Discord server that fixes that particular issue, I've pinned it in the `#troubleshooting` channel: https://discord.gg/musicpresence

**Comment by ungive** [comment]
I just released 2.2.6 which fixes Apple Music for some people on Apple Silicon Macs which had missing cover images. Please give it a try and let me know if it fixes it for you!

**Comment by HJ2343** [comment]
This did it!  Thanks!
